dotcom-rendering/src/web/components/MainMedia.tsx

```diff
@@ -48,15 +48,22 @@ const immersiveWrapper = css`
     overflow: hidden;
 `;
 
-const chooseWrapper = (format: ArticleFormat) => {
+const chooseWrapper = (
+	format: ArticleFormat,
+	showMediaAboveHeadline?: boolean,
+) => {
 	switch (format.display) {
 		case ArticleDisplay.Immersive:
 			return immersiveWrapper;
 		case ArticleDisplay.Standard: {
 			switch (format.design) {
 				case ArticleDesign.LiveBlog:
 				case ArticleDesign.DeadBlog:
-					return '';
+					if (showMediaAboveHeadline) {
+						return noGutters;
+					} else {
+						return '';
+					}
```

Comment:
As above, where possible we should avoid nesting booleans into switch statements for format if `format` already provides this data (is a live blog). I think the only decision needed here is the temporary one of if the reader is in the test bucket or not

Reply:
Good point! I'll refactor this!